static/src/javascripts/boot.js

```diff
@@ -29,6 +32,11 @@ const go = () => {
         markTime('standard boot');
         bootStandard();
 
+        // Start CMP
+        if (isInVariantSynchronous(ccpaCmpTest, 'variant')) {
+            init({ useCcpa: true });
```

Comment:
I would expect seeing CCPA parameters in there as well. At least the cookie mmsDomain and potentially others for flexibility. Changing CCPA/sourcepoint parameters should not require building/installing a new version of the consent-managamenent-platform library.

Reply:
are those parameters likely to be different across installations?

Reply:
@sndrs yes, as different installations (web properties) will have to have their own consent state. (In particular, permissionning happens at property scope). We can let that slip for now for the sake of releasing, but then, file a card to change this as it's going to be needed.

Reply:
will resolve this for a a future PR, as it doesn't affect the intention of this one